### PR TITLE
Additional i18n translations

### PR DESCRIPTION
**Pull request by laurakwhit**
## Description & motivation 💭 
Updates the following w/ translations
* `/select-namespace` 
* `/namespaces/[namespace]/archival`
* `/import/events`
* `aria-labels` on next/previous buttons in Pagination components 
* Temporal and UI version banners

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Issue(s) closed 
`DT-1324`

### CODE REVIEW

File: src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte [hunk 1]
```diff
@@ -81,28 +97,32 @@
...
+        name={translate('workflow-type')}
         value={workflowTypeFilter}
       />
-      <FilterSelect label="Time Range" parameter="time-range" value="24 hours">
+      <FilterSelect
+        label={translate('workflows', 'time-range')}
+        parameter="time-range"
+        value="24 hours"
+      >
         {#each durations as value}
```
Comment (laurakwhit) [inline src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte:113]: Have we aligned on an approach to durations yet (e.g. hrs/mins/days)?

Comment (rossedfort) [reply src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte:113]: Not yet, I think we'll need to refactor some components a little, as I don't think we can call `translate()` outside of a svelte component.

**Review by rossedfort — APPROVED** [review]

File: src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte:113]: like the approach to translate them?

Comment (rossedfort) [reply src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte:113]: Yeah, we might have to make these i18n keys, https://github.com/temporalio/ui/blob/main/src/lib/utilities/to-duration.ts#L7-L37 but then there are other utils that use those files, so they'd need to be refactored I think. There might be a simpler way to do it though

**Pull request merged**